app/views/finders/show_all_content_finder.html.erb

```diff
@@ -14,7 +14,16 @@
 <% end %>
 <% content_for :meta_title, content_item.title %>
 
-<%= render 'govuk_publishing_components/components/contextual_breadcrumbs', content_item: content_item.as_hash %>
+<% if @breadcrumbs.breadcrumbs %>
```

Comment:
Just a thought after approving. Is @breadcrumbs always an object? Just wondered if this could return nil and you get a NoMethodError

Reply:
Yes - it comes from [`FindersController`](https://github.com/alphagov/finder-frontend/blob/main/app/controllers/finders_controller.rb#L162) and is always set. Just the return value of `#breadcrumbs` may be nil which is a bit awkward, but I didn't want to start refactoring that as I'd probably go down a bit of a rabbithole!